plugins/home/src/homePageComponents/QuickStart/styles.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+/** @public */
+export type QuickStartCardClassKey =
+  | 'link'
+  | 'linkText'
+  | 'contentActionContainer'
+  | 'cardTitleIcon'
+  | 'videoContainer'
+  | 'imageSize'
+  | 'contentModal';
+
+export const useStyles = makeStyles(
+  (theme: Theme) => ({
+    link: {
+      display: 'inline-flex',
+      alignItems: 'center',
+      textDecoration: 'none',
+      color: `${theme.palette.link}!important`,
```

Comment:
do we need to use the important flag here? would love to avoid it if possible

Reply:
sorry about that! All the style changes make sense. I should have self reviewed better!